src/utils/json.cpp

```diff
@@ -135,4 +140,13 @@ Poco::Dynamic::Var FindByPath(const Poco::Dynamic::Var object, const std::vector
     return result;
 }
 
+std::string Stringify(const Poco::Dynamic::Var& json)
```

Comment:
I see this usage in our code: https://github.com/aosedge/aos_core_iam_cpp/blob/a636b224d118cfb7dfb473ca98f3e5c634a62dc5/src/database/database.cpp#L23

Will it be better to have template here to cover more cases?

Reply:
I see no benefits in making it a template right now.
[Var](https://docs.pocoproject.org/current/Poco.Dynamic.Var.html) allows to store data of different types and to convert between these types transparently.

BTW, Poco::JSON accepts Poco::Dynamic::Var
static void stringify(
    const [Dynamic::Var](https://docs.pocoproject.org/current/Poco.Dynamic.Var.html) & any,